### PR TITLE
Load NMR Data

### PR DESCRIPTION
**Pull request by Smn-Wltr**

### CODE REVIEW

**Comment by JCZuurmond** [comment]
Could you make the `Load_NMR` file a `Load_nmr.py` file?

File: Load_Onion_NMR [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import pandas as pd
+import scipy.io as scipy
+
+def load_data(path_mat):
+    onion_nmr = scipy.loadmat('../onion_NMR.mat')
```
Comment (rbschoop) [inline Load_Onion_NMR]: could you rename '../onion_NMR.mat' to path_mat

**Comment by JCZuurmond** [comment]
And could you update your branch with the master, so that we have the updates Rotraut made?

(tip: pull in the master branch in VScode while being in this `Smn-Wltr-patch-1` branch)

**Comment by Smn-Wltr** [comment]
Uploaded

**Pull request closed without merging**